node/heartbeat/sender.go

```diff
@@ -15,6 +15,7 @@ type Sender struct {
 	privKey       crypto.PrivateKey
 	marshalizer   marshal.Marshalizer
 	topic         string
+	shardID       uint32
```

Comment:
this should have been a sharding.Coordinator instance instead of the shardId

Reply:
done